app/src/main/java/com/example/pantrypal/NewRecipe.java

```diff
@@ -28,6 +33,12 @@ public class NewRecipe extends AppCompatActivity {
     private DatabaseReference databaseReference;
     private CreateRecipe createRecipe;
     private ProgressBar progressBar;
+    private TextView tagsBox;
+    private String author;
+
+    String name = "";
+    String[] ingredients = {};
+    String[] instructions = {};
```

Comment:
Could you add `private` before these String declarations?

Reply:
done